extensions/notebook/package.json

```diff
@@ -154,6 +162,15 @@
           "light": "resources/light/save.svg"
         }
       },
+      {
+        "command": "notebook.command.trustBook",
```

Comment:
Also - there currently doesn't appear to be a way to untrust a book (without modifying the config manually)? Is that by design?

Reply:
That's correct. The design so far does not include providing an "Untrust" functionality, Let me forward you the email.

Reply:
This is a good point. I raised the same point with the designer in the email that I sent you.

Reply:
I actually like that there isn't a way to untrust a book. I think it's a much less used scenario. But that's just me 😄

Reply:
If @chlafreniere doesn't think there's a need for untrusting a book then I'd go with that. Sync up with him on this.

Reply:
Synced up with Chris. Will be removing it.